ginga/rv/plugins/ColorMapPicker.py

```diff
@@ -36,11 +38,19 @@
 __all__ = ['ColorMapPicker']
 
 
-class ColorMapPicker(GingaPlugin.GlobalPlugin):
+class ColorMapPicker(GingaPlugin.LocalPlugin):
```

Comment:
Interesting, so subclassing LocalPlugin can still act like global plugin?

Reply:
Yes, there are a couple of existing examples. Check out `FBrowser`.